### DISCUSSION TITLE
docker单机水平扩展dubbo应用，如何bind dubbo应用端口

### DISCUSSION THREAD

**Discussion by crazyxtu** [apache/dubbo#13920] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

docker（docker compose编排）部署dubbo应用，如果这个dubbo应用水平扩展的话，命令 docker compose up -d scale web=4，在docker compose中需要注释掉端口，这样才能灵活水平扩展，避免端口被占用。如果没有bind到宿主机的端口，请问这时候如何注册4个web的dubbo端口呢？

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Please re-submit the disscussion in English:)
